guides/source/getting_started.md

```diff
@@ -2353,10 +2353,11 @@ module could be used in multiple models to provide the same functionality.
 A subscriber may want to unsubscribe at some point, so let's build that next.
 
 First, we need a route for unsubscribing that will be the URL we include in
-emails.
+emails, explicitly setting the identifying param to `token` (instead of the
+standard `id`).
 
 ```ruby
-  resource :unsubscribe, only: [ :show ]
+  resource :unsubscribe, param: :token, only: [ :show ]
```

Comment:
The `param` argument has no effect here as it's a singular resource. It only works for resource**s** (plural).

Are you sure that calling `unsubscribe_url(token: "blabla")` doesn't work? It should add the token as a query string parameter (`https://example.com/unsubscribe?token=blabla`) regardless of what's in the route definition.

Reply:
You're right. This is a miss at my end as I used resources declaration in my follow along project. I'll close this as it's a non-issue. Thanks for the review!